operator/pkg/upgrade/installation.go

```diff
@@ -42,24 +49,6 @@ func CreateInstallation(ctx context.Context, cli client.Client, original *ecv1be
 	return nil
 }
 
-// reApplyInstallation updates the installation spec to match what's in the configmap used by the upgrade job.
-// This is required because the installation CRD may have been updated as part of this upgrade, and additional fields may be present now.
-func reApplyInstallation(ctx context.Context, cli client.Client, in *ecv1beta1.Installation) error {
```

Comment:
this is no longer needed as now we update the CRD before making this installation